src/testkit/scala/tools/testkit/ReflectUtil.scala

```diff
@@ -12,20 +12,28 @@
 
 package scala.tools.testkit
 
-import scala.reflect.ClassTag
+import scala.reflect.{ ClassTag, classTag, ensureAccessible }
 import scala.util.chaining._
-import java.lang.reflect.{Array => _, _}
+import java.lang.reflect.{ Array => _, _ }
```

Comment:
Maybe we should just insert tabs inside the braces.

Reply:
Let's tab-le it for now? 🤺